jupyter_sphinx/execute.py

```diff
@@ -462,6 +616,11 @@ def setup(app):
         'env',
     )
 
+    # ipywidgets config
+    require_url_default = 'https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js'
```

Comment:
I don't like having duplicate default in 1 package (this is already in embed_widgets right?)